ui/src/containers/ConfigurationView.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Row, Col } from 'patternfly-react';
+import { connect } from 'react-redux';
+import Header from '../components/configuration/Header';
+import { ServiceSDKInfo } from '../components/configuration/ServiceSDKInfo';
+import { ServiceSDKDocs } from '../components/configuration/ServiceSDKDocs';
+import { CopyToClipboardMultiline } from '../components/common/CopyToClipboardMultiline';
+
+import '../components/configuration/ConfigurationView.css';
+
+// TODO remove mock values
+
+const serviceSDKInfo = [
+  {
+    serviceLogoUrl: 'https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png',
+    serviceName: 'Identity Management',
+    serviceId: 'dh-keycloak-apb-h7k9j',
+    serviceDescription: 'Identity Management - Identity and Access Management',
+    setupText: 'Identity Management SDK setup',
+    docsLink: 'https://docs.aerogear.org/aerogear/latest/identity-management.html#setup'
+  },
+  {
+    serviceLogoUrl: 'https://avatars1.githubusercontent.com/u/3380462?s=200&v=4',
+    serviceName: 'Mobile Metrics',
+    serviceId: 'dh-metrics-apb-wqm5c',
+    serviceDescription: 'Installs a metrics service based on Prometheus and Grafana',
+    setupText: 'Mobile Metrics SDK setups',
+    docsLink: 'https://docs.aerogear.org/aerogear/latest/mobile-metrics.html#setup'
+  }
+];
+
+const ConfigurationView = props => {
+  const { status = {} } = props.app;
+  const { services = [] } = status;
+  const { metadata = {} } = props.app;
+  const config = {
+    version: 1,
+    clusterName: props.openshiftMasterUrl,
+    namespace: metadata.namespace || '',
+    clientId: metadata.name || '',
+    services: []
+  };
```

Comment:
The information is already available on the backend [here](https://github.com/aerogear/mobile-developer-console/blob/master/pkg/web/mobileClientsHandler.go#L74). Can we not update the backend to return the correct value, rather than override them on the frontend here?